mitmproxy/io/db.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+import os
+
+from mitmproxy.io import protobuf
+
+
+class DbHandler:
```

Comment:
A tiny point, but can we name this DBHandler, please. It's a borderline case, but I think consistent caps in DB is less surprising.

Reply:
Oooops. That was unintended-persistent typo, DB is the way to go.